pyramid_oereb/contrib/print_proxy/mapfish_print/mapfish_print.py

```diff
@@ -497,16 +497,17 @@ def convert_to_printable_extract(self, extract_dict, feature_geometry):
         extract_dict['RealEstate_RestrictionOnLandownership'] = restrictions
         # End one restriction entry per theme
 
-        for i, item in enumerate(extract_dict.get('Disclaimer', [])):
+        # Flatten the disclaimer dictionary
+        for item in extract_dict.get('Disclaimer', []):
             self._multilingual_text(item, 'Title')
             self._multilingual_text(item, 'Content')
 
-            if i == 0:
-                extract_dict['DisclaimerLandRegister'] = item
-                self._flatten_object(extract_dict, 'DisclaimerLandRegister')
-            if i == 1:
-                extract_dict['DisclaimerPollutedSites'] = item
-                self._flatten_object(extract_dict, 'DisclaimerPollutedSites')
+        # Separate the first item, so it can be placed in a different column in the template
+        disclaimer = extract_dict.get('Disclaimer', [])
+        if len(disclaimer) > 0:
```

Comment:
does it also work with 0 disclaimers ?
Wouldn't it be good to add a test with 0 disclaimers, if there is time for it ?

Reply:
The print specs will be generated with 0 disclaimers but the MFP part would crash because it expects something in the templates.
From what I understand there should always be a min of 2 disclaimers. The reason why I added the check for `> 0` is because there seems to be an issue with this test: [test_mfp_servicea](https://github.com/openoereb/pyramid_oereb/blob/503d396835aca8c3a0e6f308a7fe9557050d11db/tests/contrib.print_proxy.mapfish_print/test_mapfish_print.py#L830-L834). I think this does not return any data.
(The reason it passed before is because it was done with a `for`.)

I agree that more tests will be useful. I'll add some :+1: